src/components/EntityInfo/EntityMenu.vue

```diff
@@ -19,50 +19,48 @@
-            <v-icon color="orange darken-2" size="24px" class="pr-2" v-on="on">mdi-alert</v-icon>
-          </template>
-          You cannot view or change business information while the business is pending dissolution.
-        </v-tooltip>
-      </span>
-
-      <!-- Dissolve Company -->
-      <span v-if="isAllowed(AllowableActions.DISSOLVE_COMPANY)">
-        <v-tooltip top content-class="top-tooltip" transition="fade-transition">
-          <template v-slot:activator="{ on }">
-            <v-btn
-              small text color="primary"
-              id="dissolution-button"
-              :disabled="hasBlocker"
-              @click="promptDissolve()"
-              v-on="on"
-            >
-              <img src="@/assets/images/Dissolution_Header_Icon.svg" alt="" class="pa-1">
-              <span class="font-13 ml-1">Dissolve this Business</span>
-            </v-btn>
-          </template>
-          Dissolving the business will make this business historical
-          and it will be struck from the corporate registry.
-        </v-tooltip>
-      </span>
-    </template>
+    <!-- View and Change Business Information -->
+    <span v-if="isBusiness && !isHistorical">
+      <v-btn
+        small text color="primary"
+        id="company-information-button"
+        :disabled="!isAllowed(AllowableActions.BUSINESS_INFORMATION)"
+        @click="promptChangeCompanyInfo()"
+      >
+        <v-icon medium>mdi-file-document-edit-outline</v-icon>
+        <span class="font-13 ml-1">View and Change Business Information</span>
+      </v-btn>
+
+      <v-tooltip v-if="isPendingDissolution" top content-class="top-tooltip" transition="fade-transition">
+        <template v-slot:activator="{ on }">
+          <v-icon color="orange darken-2" size="24px" class="pr-2" v-on="on">mdi-alert</v-icon>
+        </template>
+        You cannot view or change business information while the business is pending dissolution.
+      </v-tooltip>
+    </span>
+
+    <!-- Dissolve Business -->
+    <span v-if="isBusiness && !isHistorical">
+      <v-tooltip top content-class="top-tooltip" transition="fade-transition">
+        <template v-slot:activator="{ on }">
+          <v-btn
+            small text color="primary"
+            id="dissolution-button"
+            :disabled="!isAllowed(AllowableActions.VOLUNTARY_DISSOLUTION)"
+            @click="promptDissolve()"
+            v-on="on"
+          >
+            <img src="@/assets/images/Dissolution_Header_Icon.svg" alt="" class="pa-1">
+            <span class="font-13 ml-1">Dissolve this Business</span>
+          </v-btn>
+        </template>
+        Dissolving the business will make this business historical
+        and it will be struck from the corporate registry.
+      </v-tooltip>
+    </span>
 
     <!-- Download Business Summary -->
-    <span v-if="isAllowed(AllowableActions.DOWNLOAD_BUSINESS_SUMMARY)">
+    <span v-if="isAllowed(AllowableActions.BUSINESS_SUMMARY)">
```

Comment:
This is a show/no-show item (unlike above where it's show/no-show and enabled/disabled).

The API should only allow these when appropriate.